src/main/java/edu/wpi/first/pathweaver/CreateProjectController.java

```diff
@@ -78,6 +124,38 @@ public Game fromString(String string) {
       }
     });
 
+    length.getItems().addAll(PathUnits.LENGTHS);
+    length.getSelectionModel().selectFirst();
+    length.setConverter(new StringConverter<>() {
+      @Override
+      public String toString(Unit<Length> object) {
+        return object.getName();
+      }
+
+      @Override
+      public Unit<Length> fromString(String string) {
+        throw new UnsupportedOperationException();
+      }
+    });
+
+    var lengthUnit = EasyBind.monadic(length.getSelectionModel().selectedItemProperty());
+    timeControls.forEach(control -> control.tooltipProperty().setValue(new Tooltip("Time delta between points")));
+    velocityControls.forEach(control -> control.tooltipProperty()
+        .setValue(new Tooltip("The maximum velocity your robot can travel.")));
+    velocityUnits.textProperty().bind(lengthUnit.map(PathUnits::velocity));
+    accelerationControls.forEach(control -> control.tooltipProperty()
+        .setValue(new Tooltip("The maximum capable acceleration of your robot.")));
+    accelerationUnits.textProperty().bind(lengthUnit.map(PathUnits::acceleration));
+    jerkControls.forEach(control -> control.tooltipProperty()
+        .setValue(new Tooltip("The maximum jerk to use when calculating motion profiles."
+            + " This is user preference.")));
+    jerkUnits.textProperty().bind(lengthUnit.map(PathUnits::jerk));
+    wheelBaseControls.forEach(control -> control.tooltipProperty()
+        .setValue(new Tooltip("Distance between the left and right of the wheel base.")));
+    wheelBaseUnits.textProperty().bind(lengthUnit.map(SimpleUnitFormat.getInstance()::format));
+
+    allTooltipControls.forEach(control -> control.tooltipProperty().get().setShowDelay(new Duration(300)));
```

Comment:
Use `control.getTooltip()` instead of directly calling the property's `get()` method
The static factory methods in `Duration` should be used instead of the constructor, eg `Duration.millis(300)` instead of `new Duration(300)`